test/Test/ORM/QueryBuilderMockerTest.php

```diff
@@ -108,6 +108,28 @@ public function testEmptyGetSingleResultIsNull()
         $this->assertNull($qb->getQuery()->getSingleResult());
     }
 
+    public function testGetArrayResultReturnsValue()
+    {
+        $qbm = new QueryBuilderMocker($this);
+        $qbm->getQuery()
+            ->getArrayResult([1, 2, 3]);
```

Comment:
Use the ugly `array(...)` syntax here and on line 119. I'm still supporting PHP 5.3 on this for the time being.